src/content/blog/2024/fastquorum_intro.mdx

```diff
@@ -0,0 +1,215 @@
+
+Importantly, this _R&D version_ of the second phase allows users to test various tool-level parameters, to optimize them for their data.
+
+The second phase also has a _High-Throughput_ version, for when performance and throughput take precedence over flexibility.
+This version consensus calls and filters in one step, thereby reducing the number of consensus reads that need to be aligned as well as reducing the number of files that are written to disk.
+
+# Release Early, Release Often
+
+... and listen to your nf-core customers.
+
+With the help of some very responsive nf-core maintainers and members, [`nf-core/fastquorum`][nf-core-fastquorum-link] had its [first official release][nf-core-fastquorum-first-release-link] at the [Nextflow Summit in Boston (2024)][nextflow-summit-boston-2024-link].
+
+It supports single-strand and duplex sequencing data, both the R&D and high-throughput fgbio best practices, combining FASTQs across runs and lanes, as well as flexible support for molecular barcoding or UMI schemes.
+
+For more documentation, head over to the [`nf-core/fastquorum`][nf-core-fastquorum-link] page, visit the [`fgbio` toolkit homepage][fgbio-homepage-link] and [`fgbio` wiki][fgbio-wiki-link], and join us on our [Slack Channel][nf-core-fastquorum-slack-link].
+
+# It Takes a Nextflow Village
+
+A number of organizations have sponsored and contributed to the development of the [`fgbio` toolkit][fgbio-github-link] including [Fulcrum Genomics][fulcrum-genomics-link], [TwinStrand Biosciences][twinstrand-biosciences-link], and [Integrated DNA Technologies][idt-link].
+Both [Fulcrum Genomics][fulcrum-genomics-link] and its [current and past team members][fulcrum-genomics-about-link], along with the nf-core community of maintainers, core team, and contributors have enabled [`nf-core/fastquorum`][nf-core-fastquorum-link]'s first release.
+
+| Fulcrum Genomics | TwinStrand Biosciences | `nf-core` Community |
+| ---------------- | ---------------------- | ------------------- |
+| Nils Homer       | Michael Hipp           | Simon Pearce        |
+| Tim Fennell      | John McGuigan          | Adam Talbot         |
+| Clint Valentine  | Thomas Smith           | Chad Young          |
+| Yossi Farjoun    | Robert N. Azad         | Peter Hickey        |
+| Jay Carey        |                        | Brad Langhorst      |
+| Kari Stromhaug   |                        | Jordi Camps         |
+| Nathan Roach     |                        | Brent Pedersen      |
+
+If you want to hear this all over again, please check out the recent talk announcing the first release of this pipeline at the Nextflow Summit in Boston (2024):
+
+    <YouTube id="https://www.youtube.com/watch?v=qW4kgr3x9Mo" poster='https://img.youtube.com/vi/qW4kgr3x9Mo/maxresdefault.jpg'/> <YouTube id="https://www.youtube.com/watch?v=qW4kgr3x9Mo" poster='https://img.youtube.com/vi/qW4kgr3x9Mo/maxresdefault.jpg' />
```

Comment:
```suggestion
    <YouTube id="https://www.youtube.com/watch?v=qW4kgr3x9Mo" poster='https://img.youtube.com/vi/qW4kgr3x9Mo/maxresdefault.jpg'/>
```

Reply:
That was the main issue for me.
Thanks @jfy133